deploy/devdeploy.py

```diff
@@ -21,13 +23,43 @@ def _setup_database(cur, database: str, schema: str, stage: str):
     )
 
 
+def _make_crud_zip(source: str, dest: str):
```

Comment:
this makes me think its time to switch to a makefile...not now but this script is becoming complex

Reply:
Wrote it down in #234 to not forget for later, will spin into a new issue when we're done with this (I expect we'll have more stuff fall out that we want to do later)